che-start-workspace/__zabbix-process-load.sh

```diff
@@ -20,10 +20,10 @@ VAL_MAX=${VALUES[5]}
 
 echo "$ZABBIX_HOST $METRIC_PREFIX-failed $ZABBIX_TIMESTAMP $VAL_FAIL"
 echo "$ZABBIX_HOST $METRIC_PREFIX-fail_rate $ZABBIX_TIMESTAMP $VAL_FAIL_RATE"
-echo "$ZABBIX_HOST $METRIC_PREFIX-average $ZABBIX_TIMESTAMP $VAL_AVG"
-echo "$ZABBIX_HOST $METRIC_PREFIX-median $ZABBIX_TIMESTAMP $VAL_MED"
-echo "$ZABBIX_HOST $METRIC_PREFIX-min $ZABBIX_TIMESTAMP $VAL_MIN"
-echo "$ZABBIX_HOST $METRIC_PREFIX-max $ZABBIX_TIMESTAMP $VAL_MAX"
+echo "$ZABBIX_HOST $METRIC_PREFIX-rt_average $ZABBIX_TIMESTAMP $VAL_AVG"
+echo "$ZABBIX_HOST $METRIC_PREFIX-rt_median $ZABBIX_TIMESTAMP $VAL_MED"
+echo "$ZABBIX_HOST $METRIC_PREFIX-rt_min $ZABBIX_TIMESTAMP $VAL_MIN"
+echo "$ZABBIX_HOST $METRIC_PREFIX-rt_max $ZABBIX_TIMESTAMP $VAL_MAX"
```

Comment:
Won't it influence the code in run.sh when generating result file? And what does this "rt" mean?
E.g.:
filterZabbixValue $ZABBIX_LOG "startWorkspace-max" "@@START_WORKSPACE_MAX@@" $RESULTS_FILE;

Reply:
Hi @Katka92, you are right, it does influence the code in run.sh, good catch :)
I'll fix it.

The RT is the abbreviation for the "response time".